pkg/TIPA.py

```
@@ -50,85 +48,110 @@ class ProcessManager(threading.Thread):
     Communicates with the GUI via a queue.
     Recieves instructions from the GUI via a different queue.
     '''
-    def __init__(self, gui_queue, command_queue):
-        super(ProcessManager, self).__init__()
+    def __init__(self, gui_queue: Queue, command_queue: Queue):
```

Ruff (0.8.0)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)